proto/api/v1/cluster_init_service.proto

```diff
@@ -63,6 +64,13 @@ message CRSGenRequest {
   string name = 1;
 }
 
+message CRSGenRequestExtended {
+  string name = 1;
+  google.protobuf.Timestamp valid_until = 2;
+  google.protobuf.Duration valid_for = 3;
+  int32 max_registrations = 4; // Support for this is to be implemented in ROX-26769.
```

Comment:
Won't this lead to a similar issue, where in the future `roxctl` will have to deal with two versions of `CRSGenRequestExtended`, one that knows about `max_registrations` and one that doesn't?